layout/SignUp/components/SignUpForm/index.tsx

```diff
@@ -137,6 +143,19 @@ export default function SignUpForm({ courses }) {
             updateScanning(!scanning);
           }}
         />
+        <Checkbox
+          selected={termsAccepted}
+          onChange={(e) => setTermsAccepted(!termsAccepted)}
+        >
+          I have read and understood the &nbsp;
+          <Link href="/docs/privacy_policy.pdf" className="text-quinary">
+            privacy policy
+          </Link>{" "}
+          and the &nbsp;
+          <Link href="/docs/regulation.pdf" className="text-quinary">
```

Comment:
To make files open in a new tab instead of replacing the current one (so that the user signing up doesn't lose the form data)
```suggestion
          <Link href="/docs/privacy_policy.pdf" target="_blank" className="text-quinary">
            privacy policy
          </Link>{" "}
          and the &nbsp;
          <Link href="/docs/regulation.pdf" target="_blank" className="text-quinary">
```